source_test.yaml

```diff
@@ -255,6 +255,18 @@
   link: 'https://github.com/RConsortium/r-advisory-database/blob/main/'
   editable: False
 
+- name: 'redhat-stage'
+  versions_from_repo: False
+  type: 0
+  ignore_patterns: ['^(?!RH[BES]{1}A-).*$']
```

Comment:
There's a discrepancy between what this matches and what https://github.com/ossf/osv-schema/pull/257 introduced...

Reply:
Those prefixes were added in https://github.com/ossf/osv-schema/pull/272, which missed the boat for https://github.com/ossf/osv-schema/releases/tag/v1.6.5